content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -714,31 +714,74 @@ Clients find resource versions in resources, including the resources in watch ev
 
 The get, list and watch operations support the `resourceVersion` parameter.
 
-The exact meaning of this parameter differs depending on the operation and the value of the resource version.
+The exact meaning of this parameter differs depending on the operation and the value of `resourceVersion`.
 
 For get and list, the semantics of resource version are:
 
 **Get:**
 
-| resourceVersion unset | resourceVersion is `0` | resourceVersion is set but not `0` |
-|-----------------------|------------------------|------------------------------------|
-| Most Recent           | Any                    | Not older than                     |
+| resourceVersion unset | resourceVersion="0" | resourceVersion="{value other than 0}" |
+|-----------------------|---------------------|----------------------------------------|
+| Most Recent           | Any                 | Not older than                         |
 
 **List:**
 
-| paging                        | resourceVersion unset | resourceVersion="0"                            | resourceVersion="{value other than 0}" |
-|-------------------------------|-----------------------|------------------------------------------------|----------------------------------------|
-| limit unset                   | Most Recent           | Any                                            | Not older than                         |
-| limit="n", continue unset     | Most Recent           | Any                                            | Exact                                  |
-| limit="n", continue="<token>" | Continue Token, Exact | Invalid, but treated as Continue Token, Exact  | Invalid, HTTP `400 Bad Request`        |
+v1.19+ API servers and newer support the `resourceVersionMatch` parameter, which
+determines how resourceVersion is applied to list calls.  It is highly
+recommended that `resourceVersionMatch` be set for list calls where
+`resourceVersion` is set. If `resourceVersion` is unset, `resourceVersionMatch`
+is not allowed.  For backward compatibility, clients must tolerate the server
+ignoring `resourceVersionMatch`:
+- When using `resourceVersionMatch=NotOlderThan` and limit is set, clients must
+  handle HTTP 410 “Gone” responses. For example, the client might retry with a
```

Comment:
outstanding “ and ”?

Reply:
oops, lost this when merged with suggested changes incorrectly. Fixed.